apps/omar-ui-app/grails-app/conf/application.yml

```diff
@@ -242,6 +242,9 @@ omar:
       enabled: true
     # Base settings for the O2 UI general configuration
     misc:
+      # types are WMS or OMS 
+      # TODO: add better discription
```

Comment:
I would address this TODO, and add a good description for what this doess.  "oms" needs the `omar-oms` service to be deployed. "wms" will use `omar-wms` or `omar-lite-wms`

Reply:
Done.